x/authz/keeper/msg_server.go

```diff
@@ -118,6 +118,17 @@ func (k Keeper) Exec(ctx context.Context, msg *authz.MsgExec) (*authz.MsgExecRes
 	return &authz.MsgExecResponse{Results: results}, nil
 }
 
+func (k Keeper) PruneExpiredGrants(ctx context.Context, msg *authz.MsgPruneExpiredGrants) (*authz.MsgPruneExpiredGrantsResponse, error) {
+	if err := k.DequeueAndDeleteExpiredGrants(ctx, 75); err != nil {
```

Comment:
nit: put this in a const

Also, was there any specific rationale for 75?

Reply:
Sure!
This one is aligned with x/feegrant, but they are arbitrary.
We were talking about adding parameters for feegrant for, but turns out that unnecessarily parametrize the module.